### DISCUSSION TITLE
“臆想症”

### DISCUSSION THREAD

**Discussion by yivitte** [share121/inter-knot#3079] [其他]
我去找心理医生
“来，孩子，坐”
“你手上有皮筋呀，哈哈，谈女朋友啦”
：“没有，我自己买的”
“自己买的呀”
：“这样可以幻想我被爱着”
“。。。（沉默）”

“恭喜你，克服了抑郁症”
“能说说你是怎么做到的吗”
：“我让自己爱上了她”
“然后呢”
：“然后她就会离开我，和我爱过的所有人一样”
![IMG_20240819_163401](https://github.com/user-attachments/assets/5e1796d3-3afb-4caf-927a-bdd75459e8a3)

**Comment by yivitte** [comment]
我初中的时候曾经谈过一场并不算轰轰烈烈的恋爱，她是个重度抑郁症患者，我是个动了恻隐之心的普通人

**Comment by yivitte** [comment]
光是接近就很困难，一不留神就会逃跑，发展到能够好好坐在一起说话就已经花了非常久的时间

**Comment by yivitte** [comment]
但她相信着我，我也相信着她，她说她发病总在十一点之后，我就十点钟发消息检查一下，如果她没睡，就过来哄她休息

**Comment by x97-1** [comment]
想开点，可能只是她进入了深度抑郁不忍心再连累你了。<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by yivitte** [comment]
她说想试着做饭，我们一起把一个大铁锅炒的全是糊味，沾满黑黑的黏黏的东西直到无法使用了

**Comment by yivitte** [comment]
她说想出一趟门，我挑了个人少的时间带她去了她一直想去的公园，因为她怕生，只是不怕我，偶尔有人也需要我帮忙遮掩一二

**Comment by yivitte** [comment]
偶尔隔着手机，偶尔面对面，我们总能找到在一起待着的方式，不需要太多的动作，不需要太多表达的方式，我们就能明白彼此心里的想法

**Comment by yivitte** [comment]
这样的梦寐以求的日子并没有过得太久，她说想自己出门一段时间，就再也没回来过

**Comment by yivitte** [comment]
我寻找她，在大街小巷间跑来跑去的，在网上发寻人启事，在亲朋好友之间询问她的下落却一无所获

**Comment by yivitte** [comment]
人生像是失去了目标，瘫坐在家里，一会想起了甜蜜的点滴笑着，一会看到她离去的背影哭着

**Comment by yivitte** [comment]
最后，她回来了，虽然她什么也没做，但是我感觉得到，她看我的眼神不同了，我们之间隔着一层不知道什么时候多出来的墙壁，再也接近不得

**Comment by yivitte** [comment]
我像以前那样试着接近她，却再也没得到过正向的反馈，她总会因为很小的事情发怒，我不得不像是背板一样根据她喜欢的方式来讨她开心

**Comment by yivitte** [comment]
我对她的喜好在这一刻空前的了解，却并未因此感到开心，我的每一天在甜甜蜜蜜到失魂落魄到诚惶诚恐到心惊胆战的睡下，哪怕睡着了，也不敢睡得太死，怕她凌晨的消息没来得及收

**Comment by yivitte** [comment]
终于有一天，她像是终于玩腻了似得，真正的消失在了我的生活中

**Comment by yivitte** [comment]
徒劳的搜寻后，我像之前那样坐在家里，我反思自己，是不是又有哪里的小细节没有做好，温柔的她虽然感到不悦但却并没有说出来，并在这种反复的检讨中将她的形象变得越发高大

**Comment by yivitte** [comment]
最后顺理成章的没考上高中，爸爸的喝骂声，妈妈掩面的哭泣声都传达不到当时跪地的我的心里，我满脑子只有她的背影，以及之前发生过的点点滴滴

**Comment by yivitte** [comment]
我不知道太阳是怎样升起又落下的，当我回过神来的时候，已经坐在了洁白的诊室，医生笑着问我，坐在这里有没有感觉好一些

**Comment by yivitte** [comment]
我头一次注意到了周围温暖的装潢，以及医生桌边的香薰，被挪到我也能看到的方向的电脑屏幕和大概是从父母那里了解到的我之前最喜欢玩的游戏

**Comment by yivitte** [comment]
我难得的在和医生玩游戏的时间中开心了一回，现在回忆起来，医生和我一起玩的时候大抵是笑着的吧

**Comment by yivitte** [comment]
她问我，是不是和女朋友分手了？

**Comment by yivitte** [comment]
我说是，她把手轻轻放在我的手背上，和我讨论了一些恋爱中的烦恼，鼓励我努力去追寻自己的喜欢，哪怕是失败，也要给自己的心一个交代

**Comment by yivitte** [comment]
可能是不经意的，她问我，你女朋友长得好不好看？

**Comment by yivitte** [comment]
我绞尽脑汁去回忆她的美貌，并将学过的最好的辞藻讲述给面前的医生

**Comment by yivitte** [comment]
她笑眯眯的看着我，在我说累了，喝了一口水休息的时候，她又问我

**Comment by yivitte** [comment]
“这么好看的姑娘，名字一定很好听吧？她叫什么名字？”

**Comment by yivitte** [comment]
我沉默了，我在这个时候开始绞尽脑汁回忆名字。但我发现我根本没有问过她名字

**Comment by yivitte** [comment]
可能是隐蔽着的枷锁被机缘巧合的方式打开了，所有之前有关她的画面都快速地在我眼前回放

**Comment by yivitte** [comment]
但是画面里哪有她的身影？只有一个手机屏幕，还有我和她的聊天记录！

**Comment by yivitte** [comment]
我的脸颊，我的背都开始疯狂的出冷汗，我怎么可能没见过她？那我之前的记忆算什么？都是假的吗

**Comment by yivitte** [comment]
我不知道。。我写到这里的时候我也感到迷茫

**Comment by yivitte** [comment]
我当时是怎么回复正常的？

**Comment by yivitte** [comment]
想不起来，记忆好像截断了，有了断层，能想到的就是我回到了我自己的房间，手里的手机也换了新的

**Comment by yivitte** [comment]
我在家里休息了几天，跟爸妈提出说我想学学计算机，因为它高薪，我也很感兴趣

**Comment by yitan1741** [comment]
哥们也喜欢过一个轻度抑郁症的女生，当然啊，没走到一起，因为这事再加上其他的事，我也是三年没走出来，浑浑噩噩了三年。
不过最后我想清楚了，即使那段记忆再美好，该走了。虽然还是不甘心吧。

其实很简单，朋友，大脑为了保护你，会让你忘掉让你感到特别痛苦的一段回忆，可能多可能少，像我前三年的记忆基本上一点都没有，回想起来就是空白。很正常的，毕竟大脑为了你，可是拼尽了一切的，忘掉更好。

**Comment by yivitte** [comment]
爸妈慷慨的支持了我的决定，只可惜，学习的成果并不算好，只是让我对电脑更了解了些，至于学习代码的事，不怕你们笑话，已经全还给老师了

**Reply by yitan1741** [reply]
不瞒您说，我的朋友，我现在也在从头开始学代码，学计算机，大学父母给我报了一个很差的专业，也没有尊重我的意见，很不甘心吧。虽然知道自己和别人已经差了很远，但想学嘛，就去尝试。

朋友，虽然可能说这些话有些马后炮的感觉，我也没办法，希望您能找到自己的生活，再多的劝勉也不如时间的抚平，我相信您一定能找到自己的路，毕竟每个人的路都是不同的，我无法指导您什么，但您要相信，您一定可以决定自己的路，一定不要放弃呀，说不定她就在未来等你

**Comment by yivitte** [comment]
现在，我拿自己打工的工资租了个房子，天气太热了，有的客户不想多花时间把车开到停车场，想进去吹空调，我放他车胎的气被开除了

**Comment by yivitte** [comment]
打算休息几天，然后看到了这个叫臆想症的视频，和大家分享了些我的个人经历

**Comment by yivitte** [comment]
镇楼图上面的是我在视频里看来的，我在评论里写的是我自己的

**Comment by yivitte** [comment]
虽然日出的结局总是日落，但黑夜的尽头也总是黎明。很抱歉，没能给可能同病相怜的朋友带来一些有用的东西，只能消耗一些你们茶余饭后的闲暇时间

**Comment by yivitte** [comment]
我并没有因为这个事情继续抑郁下去，可能是忘了就好了吧，我现在没有再复发过臆想症这种事情了

**Comment by yivitte** [comment]
现在仍然能抱着善良的心迎接着社会，生活已经很苦了，何必再给自己添麻烦？是吧

**Comment by yivitte** [comment]
承蒙你的关心，我现在过的挺好的，偶尔有不顺心，但大部分时候很快乐



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月19日 17:23 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] “臆想症” (Discussion #3079) |

不瞒您说，我的朋友，我现在也在从头开始学代码，学计算机，大学父母给我报了一个很差的专业，也没有尊重我的意见，很不甘心吧。虽然知道自己和别人已经差了很远，但想学嘛，就去尝试。

朋友，虽然可能说这些话有些马后炮的感觉，我也没办法，希望您能找到自己的生活，再多的劝勉也不如时间的抚平，我相信您一定能找到自己的路，毕竟每个人的路都是不同的，我无法指导您什么，但您要相信，您一定可以决定自己的路，一定不要放弃呀，说不定她就在未来等你
